spec/javascripts/manifest.js

```diff
@@ -15,12 +15,12 @@ var manifest = {
     '../../../app/assets/javascripts/analytics/_scrollTracking.js',
     '../../../app/assets/javascripts/analytics/_init.js'
   ],
-  test : [
+  test: [
     '../unit/AnalyticsSpec.js',
-    '../unit/piiSpec.js',
+    '../unit/piiSpec.js'
```

Comment:
These paths have been changed, haven't they? Is this `manifest.js` file still used?

Reply:
Yes, they have changed. `manifest.js` is not needed anymore by I left this file in because I think we might need to refer to it when we come to re-adding the tests back in.

Reply:
I say ditch it. There aren't that many test files!

Reply:
Just binned it 👍